### PR TITLE
Localize css js

### PR DESCRIPTION
**Pull request by dzr201732120115**
* **What kind of change does this PR introduce?**
**optimize**.
Localize css and js file,which allows us to refer to them from our site's address.
Because sometimes cdnjs can be slowly or inaccessible.

* **What is the current behavior?** 
The site references the CSS/JS files from the CDNJS site. 
such as: [https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js](url)

* **What is the new behavior 
The site references the CSS/JS files from itself.
such as: [https://kubeedge.io/en/cdnjs.cloudflare.com/jquery.min.js](url)

* **Does this PR introduce a breaking change?**
No,it just cover origin layouts file,such as footer.html.So if you want to restore it,you just need to delete the new files.

* **Other information**:
The new files (the same as assets.toml):
> /website/layouts/partials/footer.html 
/website/layouts/partials/header.html
will cover 
>/website/themes/academic/layouts/partials/footer.html 
/website/themes/academic/layouts/partials/header.html 
So it won't influence the theme academic 's update.

### CODE REVIEW

**Comment by daixiang0** [comment]
Could share screenshot or demo link about new layout?

I think you can use GITHUB PAGE as demo link :)

File: data/assets.toml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Versioning and Subresource Integrity (SRI) for Academic's JavaScript and CSS dependencies
+
+# When updating the version of an asset below, please also update the corresponding SRI.
+# How to update the SRI for an entry:
+#   echo -n "sha512-" && curl --silent "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS_CHTML" | openssl dgst -sha512 -binary | openssl enc -base64 -A
+# Or, the SRI for the appropriate asset can be copied from https://cdnjs.com/ .
+
+# JavaScript
+
```
Comment (kevin-wangzefeng) [inline data/assets.toml:9]: Please add NOTE to mark where is this file from, just in case we may need to update some of the js versions later on.

**Comment by dzr201732120115** [comment]
[https://testke.netlify.app/en/](url)
I just modify the link from cdnjs to local.So page layout won't change.
localizeCssJs

File: data/assets.toml (hunk 1 above)
Comment (dzr201732120115) [reply data/assets.toml:9]: > \#  version = "3.3.1"
> \#  url = "https://cdnjs.cloudflare.com/ajax/libs/jquery/%s/jquery.min.js"

We just need to replace "%s" with version's data.  (don't manage sri)
For example,we'd like to change jquery's version from 3.3.1 to 4.0.1(suppose exist).
Then:
1. Access to: [https://cdnjs.cloudflare.com/ajax/libs/jquery/4.0.1/jquery.min.js"](url)
2. Right click save as. (don't copy all and paste,could causes a newline problem)
3. Replace origin jquery.min.js file in the /website/static/cdnjs.cloudflare.com/jquery.min.js

By the way,too new versions could be unstable.

Comment (daixiang0) [reply data/assets.toml:9]: It would be better to provide a tool to update those js files.

**Comment by daixiang0** [comment]
I just find that we can also visit links in the details of pr check, while it is slow. It is very cool to have a demo website we can visit.

**Comment by dzr201732120115** [comment]
> I just find that we can also visit links in the details of pr check, while it is slow. It is very cool to have a demo website we can visit.
https://testke.netlify.app/en/

**Comment by dzr201732120115** [comment]
> > I just find that we can also visit links in the details of pr check, while it is slow. It is very cool to have a demo website we can visit.
> > https://testke.netlify.app/en/

**Comment by kevin-wangzefeng** [comment]
Thanks @dzr201732120115,@daixiang0, to me this PR bound static resources together with the main site assets, all resources would be accessable as long as kubeedge.io is available.
To improve the page loading speed, we will still need global available CDN to help.

**Comment by kevin-wangzefeng** [comment]
Accepting the PR for now. We can simply remove "assets.toml" to use original CDN if we want.

**Pull request merged**